src/timer.cr

```diff
@@ -0,0 +1,28 @@
+module SDL
+  module Timer
+    
+    def self.add(interval, &callback : UInt32 -> Void)
+      LibSDL.add_timer(interval, callback)
+    end
+
+    def self.delay(ms : UInt32)
+      LibSDL.delay(ms)
```

Comment:
`SDL_Delay()` duplicates `sleep()` and AFAIK block the current thread, thus block the crystal event loop.